CMakeLists.txt

```diff
@@ -341,7 +341,7 @@ add_subdirectory(iked)
 add_subdirectory(ikectl)
 add_subdirectory(regress/dh)
 add_subdirectory(regress/parser)
-if(CMAKE_SYSTEM_NAME MATCHES "Linux" AND CMAKE_C_COMPILER_ID STREQUAL "Clang")
```

Comment:
Why isn't this needed anymore? Isn't libfuzzer part of clang/llvm?

Reply:
Libfuzzer is part of clang/llvm, but clang is not needed as compiler for the whole project.
Clang is defined as compiler only for parser-libfuzzer in regress/parser-libfuzzer/CMakeLists.txt:15

Reply:
Makes sense, but without this check the build will probably fail if llvm is not installed. What this check should actually do is see if clang + libfuzzer is installed on the system. The current check is not perfect but works because if you are compiling with clang then llvm + libfuzzer must be installed.

Reply:
Ah, now I see the problem. Ok, I'll think of an option to check if clang/llvm is installed.

The problem with the old check is, that it is only true, if the default compiler is already clang, which is for example not the case on my fedora-machine.